### ISSUE TITLE
Can't get through configuration on launch

### ISSUE THREAD

**Issue by nilsborg** [kactus-io/kactus#27]
### Description

Hi there! I'm super excited to try Kactus. Downloaded it from the website, unzipped it, moved to Apps, opened it and now tried to get through the wizard von first launch.

I skipped the first question if I'd like to sign in with Github (actually also did that later but didn't change anything). Wanted to click continue on the screen with my name and email but it's not doing anything …

![kactus](https://user-images.githubusercontent.com/1119139/28459508-0dac2f98-6e0e-11e7-8cd3-5fc912ec61c2.gif)

### Version 

**Kactus version:** 0.1.11

**OS version:** Mac OS 10.12.5 (16F73)

### Logs

Sadly no logs …

**Comment by nilsborg** [comment]
Ah yes so this is what I get when i open Console.app:

```
default	12:14:52.677617 +0200	Kactus Helper	GVA warning: failed to get a service for display id: 4128828
default	12:14:52.691123 +0200	Kactus Helper	GVA info: preferred scaler idx 0
default	12:14:52.950647 +0200	Kactus	UNIX error exception: 17
default	12:14:52.951048 +0200	Kactus	0x600000271680 opened /private/var/db/mds/system/mdsDirectory.db: 50744 bytes
default	12:14:52.952053 +0200	Kactus	UNIX error exception: 17
default	12:14:53.018388 +0200	Kactus	UNIX error exception: 17
default	12:14:53.018520 +0200	Kactus	UNIX error exception: 17
default	12:14:53.018634 +0200	Kactus	0x6100004693c0 opened /System/Library/Keychains/SystemRootCertificates.keychain: 385940 bytes
default	12:14:53.022754 +0200	Kactus	Invalid serial number length (21), must be 1..20
default	12:14:53.031144 +0200	Kactus	UNIX error exception: 17
default	12:14:53.273209 +0200	Kactus	Faulting in NSHTTPCookieStorage singleton
default	12:14:53.273233 +0200	Kactus	Faulting in CFHTTPCookieStorage singleton
default	12:14:53.273245 +0200	Kactus	Creating default cookie storage with process/bundle identifier
default	12:14:53.274924 +0200	Kactus	UNIX error exception: 17
default	12:14:53.275108 +0200	Kactus	UNIX error exception: 17
default	12:14:53.275242 +0200	Kactus	UNIX error exception: 17
default	12:14:53.275461 +0200	Kactus	0x618000461900 opened /Users/nilsborchers/Library/Keychains/login.keychain-db: 1743656 bytes
default	12:14:53.279802 +0200	Kactus	0x600000276a00 opened /Library/Keychains/System.keychain: 171428 bytes
default	12:14:53.283049 +0200	Kactus	TIC TCP Conn Start [1:0x61000019b5f0]
default	12:14:53.419037 +0200	Kactus	TIC TCP Conn Event [1:0x61000019b5f0]: 1 Err(0)
default	12:14:53.419074 +0200	Kactus	TIC TCP Conn Connected [1:0x61000019b5f0]: Err(0)
default	12:14:53.419205 +0200	Kactus	TIC Enabling TLS [1:0x61000019b5f0]
default	12:14:53.533570 +0200	Kactus	TIC TLS Event [1:0x61000019b5f0]: 2, Pending(0)
default	12:14:53.533781 +0200	Kactus	TIC TLS Event [1:0x61000019b5f0]: 11, Pending(0)
default	12:14:53.534253 +0200	Kactus	TIC TLS Event [1:0x61000019b5f0]: 12, Pending(0)
default	12:14:53.534284 +0200	Kactus	TIC TLS Event [1:0x61000019b5f0]: 14, Pending(0)
default	12:14:53.553193 +0200	Kactus	TIC TLS Trust Result [1:0x61000019b5f0]: 0
default	12:14:53.666518 +0200	Kactus	TIC TLS Event [1:0x61000019b5f0]: 20, Pending(0)
default	12:14:53.666844 +0200	Kactus	TIC TCP Conn Event [1:0x61000019b5f0]: 8 Err(0)
default	12:14:53.666871 +0200	Kactus	TIC TLS Handshake Complete [1:0x61000019b5f0]
```

**Comment by rslotman** [comment]
Same problem here. Is there a fix?

**Comment by therephil** [comment]
Same Problem here. 

Here is how it worked for me:
Install the [previous version](https://github.com/kactus-io/kactus/releases/tag/v0.1.10) first. Then kaktus can update itself.

This is obviously not the best onboarding experience but works to try it out

**Comment by nilsborg** [comment]
🤙🏼
cool thanks!

**Comment by mathieudutour** [comment]
Oops, embarrassing. Fixed in 0.1.12 (deploying now)
